modules/registry-configuring-storage-nutanix.adoc

```diff
@@ -6,4 +9,4 @@
 
 After you install the cluster, you must install the Nutanix CSI Operator and configure the default storage container for the cluster.
 
-For more information, see the Nutanix documentation for link:https://opendocs.nutanix.com/openshift/operators/csi/[installing the CSI Operator] and link:https://opendocs.nutanix.com/openshift/install/ipi/#openshift-image-registry-configuration[configuring registry storage].
+For more information, see the Nutanix documentation for link:https://opendocs.nutanix.com/openshift/operators/csi/[installing the CSI Operator] and link:https://opendocs.nutanix.com/openshift/post-install/[configuring registry storage].
```

Comment:
```suggestion
See link:https://opendocs.nutanix.com/openshift/operators/csi/[Nutanix CSI Operator, Installing the Operator] and link:https://opendocs.nutanix.com/openshift/post-install/[OpenShift Image registry configuration] for details.
```
Changed to be more literal. It took me a second to figure out where I should go. Easier with the exact linked titles.

Reply:
Pretty sure our guidance requires us to use, "For more information". Going to leave as is.